docs/ipr/scripting/RNA_Expression_Metrics.ipynb

```diff
@@ -0,0 +1,4089 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "gHdydgo7Oq4N",
+        "outputId": "afa11709-9c63-4044-9271-e128c6debcb9"
+      },
+      "source": [
+        "!wget https://toil-xena-hub.s3.us-east-1.amazonaws.com/download/tcga_RSEM_gene_tpm.gz\n",
+        "!wget https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/Survival_SupplementalTable_S1_20171025_xena_sp"
+      ],
+      "execution_count": 38,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "--2021-06-09 19:24:14--  https://toil-xena-hub.s3.us-east-1.amazonaws.com/download/tcga_RSEM_gene_tpm.gz\n",
+            "Resolving toil-xena-hub.s3.us-east-1.amazonaws.com (toil-xena-hub.s3.us-east-1.amazonaws.com)... 52.216.80.45\n",
+            "Connecting to toil-xena-hub.s3.us-east-1.amazonaws.com (toil-xena-hub.s3.us-east-1.amazonaws.com)|52.216.80.45|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 740772247 (706M) [binary/octet-stream]\n",
+            "Saving to: ‘tcga_RSEM_gene_tpm.gz.1’\n",
+            "\n",
+            "tcga_RSEM_gene_tpm. 100%[===================>] 706.46M  25.4MB/s    in 24s     \n",
+            "\n",
+            "2021-06-09 19:24:38 (30.0 MB/s) - ‘tcga_RSEM_gene_tpm.gz.1’ saved [740772247/740772247]\n",
+            "\n",
+            "--2021-06-09 19:24:38--  https://tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com/download/Survival_SupplementalTable_S1_20171025_xena_sp\n",
+            "Resolving tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com (tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com)... 52.217.93.16\n",
+            "Connecting to tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com (tcga-pancan-atlas-hub.s3.us-east-1.amazonaws.com)|52.217.93.16|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 2419504 (2.3M) [binary/octet-stream]\n",
+            "Saving to: ‘Survival_SupplementalTable_S1_20171025_xena_sp.1’\n",
+            "\n",
+            "Survival_Supplement 100%[===================>]   2.31M  7.80MB/s    in 0.3s    \n",
+            "\n",
+            "2021-06-09 19:24:38 (7.80 MB/s) - ‘Survival_SupplementalTable_S1_20171025_xena_sp.1’ saved [2419504/2419504]\n",
+            "\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Ef8Dl3dXPUu1"
+      },
+      "source": [
+        "The matrix we have downloaded is very large as it contains many different disease cohorts. For the purposes of this tutorial we want a smaller dataset so we are going to choose the smallest TCGA cohort, CHOL. To do this we need to use the metadata file to subset the main matrix file"
```

Comment:
i haven't run into 'subset' as a verb before, is this intentional?

Reply:
yup! Weird. you're right when i look it up in an english dictionary but it is a pretty common science/data-science/math usage so I think we're ok